src/main.html

```diff
@@ -169,7 +169,7 @@
             <div id="portsinput">
                 <div class="dropdown dropdown-dark">
                     <select class="dropdown-select" id="port" i18n_title="firmwareFlasherManualPort">
-                        <option value="manual">Manual</option>
+                        <option value="manual">Loading</option>
```

Comment:
By default,  the list of ports have the "manual" one. If you want to change it by "Loading" until the list of ports is retrieved is better to change/remove the value too, not only the tag. I will use the localization too to make it translatable.

Reply:
Done!